src/UserBundle/DependencyInjection/Extension.php

```diff
@@ -101,6 +102,10 @@ public function load(array $configs, ContainerBuilder $container): void
                 SecurityInfra\UserParamConverter::class,
```

Comment:
you can add `SecurityInfra\Jwt\SecurityUserProvider::class` here and remove the `removeIf` call below.